hedera-node/cli-clients/src/main/java/com/hedera/services/cli/signedstate/DumpContractStoresSubcommand.java

```diff
@@ -177,8 +177,8 @@ boolean iterateThroughContractStorage(BiConsumer<ContractKey, IterableContractVa
             contractStorageVMap.extractVirtualMapData(
                     getStaticThreadManager(),
                     entry -> {
-                        final var contractKey = entry.getKey();
-                        final var iterableContractValue = entry.getValue();
+                        final var contractKey = entry.left();
```

Comment:
Just fyi there is still `key()` and `value()` available

Reply:
ok.  is there a convention on whether to you key/value or left/right I wonder?